core/src/main/resources/lib/form/entry.jelly

```diff
@@ -72,7 +72,7 @@ THE SOFTWARE.
   <!-- expose this so that we can look up the @field value later from prepareDatabinding.jelly -->
   <j:set var="entry" value="${attrs}" />
   <j:set var="possiblyEscapedTitle" value="${escapeEntryTitleAndDescription ? h.escape(attrs.title) : attrs.title}" />
-  <div class="jenkins-form-item ${attrs.class}">
+  <div class="jenkins-form-item tr ${attrs.class}">
```

Comment:
@janfaracik is it ok to restore this?

I haven't been able to track down what's missing, some notes in https://issues.jenkins.io/browse/JENKINS-67109

Reply:
Looks good to me 👍